jenkins/lava-boot-v2.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*
+#
+
+import os
+import sys
+import subprocess
+
+"""
+To test the script, just export those variables and play with their values
+
+export LAB=lab-free-electrons
+export API=https://api.kernelci.org
+export API_TOKEN=blehblehbleh
+export STORAGE=https://storage.kernelci.com/
+export CALLBACK=https://callback.api
+export GIT_DESCRIBE=v4.12
+export TREE=mainline
+export BRANCH=master
+export ARCH=arm
+
+export LAVA_TBAKER_TOKEN=ploploploploploploploplop
+export LAVA_MHART_TOKEN=ploploploploploploploplop
+export LAVA_COLLABORA_TOKEN=ploploploploploploploplop
+export LAVA_BAYLIBRE_TOKEN=ploploploploploploploplop
+export LAVA_BAYLIBRE_SEATTLE_TOKEN=ploploploploploploploplop
+export LAVA_FREE_ELECTRONS_TOKEN=ploploploploploploploplop
+"""
+
+# Reproduce the Jenkins environment for easier use.
+# Basic env
+LAB = os.environ['LAB']
+API = os.environ['API']
+API_TOKEN = os.environ['API_TOKEN']
+STORAGE = os.environ['STORAGE']
+CALLBACK = os.environ['CALLBACK']
+GIT_DESCRIBE = os.environ['GIT_DESCRIBE']
+TREE = os.environ['TREE']
+BRANCH = os.environ['BRANCH']
+ARCH = os.environ['ARCH']
+
+# Tokens
+LAVA_TBAKER_TOKEN = os.environ['LAVA_TBAKER_TOKEN']
+LAVA_MHART_TOKEN = os.environ['LAVA_MHART_TOKEN']
+LAVA_COLLABORA_TOKEN = os.environ['LAVA_COLLABORA_TOKEN']
+LAVA_BAYLIBRE_TOKEN = os.environ['LAVA_BAYLIBRE_TOKEN']
+LAVA_BAYLIBRE_SEATTLE_TOKEN = os.environ['LAVA_BAYLIBRE_SEATTLE_TOKEN']
+LAVA_FREE_ELECTRONS_TOKEN = os.environ['LAVA_FREE_ELECTRONS_TOKEN']
+
+# Config
```

Comment:
I've made a first version of `should-I-boot-it.py` without, but of course, it's possible to do that.